HTML/mockup-corporate/sass/layout/_header.scss

```diff
@@ -0,0 +1,46 @@
+.header {
+    padding: 10% 0;
+    background-image: url(img/header@3x-min.png);
+    background-size: cover;
+    background-position: center;
+    @include flex($justify: center, $align: center);
```

Comment:
I would recommend at first write mixins and after your css code

and generally same result can be archived with less code and without flexes with just text-align: center + margin: 0 auto;